fe/fe-core/src/test/java/com/starrocks/analysis/ShowGrantsStmtTest.java

```diff
@@ -68,8 +69,12 @@ public void testNormal() throws Exception {
                 result = true;
             }
         };
-        ShowGrantsStmt stmt = new ShowGrantsStmt(new UserIdentity("test_user", "localhost"), false);
-        stmt.analyze(analyzer);
+        String revokeSql = "SHOW GRANTS FOR test_user@'localhost'";
+        try {
+            UtFrameUtils.parseStmtWithNewParser(revokeSql, ctx);
```

Comment:
I suggest you throw the exception out so that when this UT fails,  other people can easily find the root cause by the shown stack.

Reply:
done